package.json

```diff
@@ -1,14 +1,12 @@
 {
   "name": "injectweb3-connect",
-  "version": "1.2.7",
+  "version": "2.0.0",
   "description": "Package for connecting wallets injected with injectWeb3 object",
-  "main": "dist/index",
-  "type": "commonjs",
+  "types": "dist/index.d.ts",
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
   "scripts": {
-    "build": "webpack --mode=production --node-env=production",
-    "build:dev": "webpack --mode=development",
-    "build:prod": "webpack --mode=production --node-env=production",
-    "watch": "webpack --watch"
+    "build": "tsup"
```

Comment:
Hey @WRadoslaw in the version `2.1.0`, `tsup` still bundles some of the `@polkadot` dependencies (although it tree shakes them pretty well). But it bundles them from their `package.json` `main` entries rather than `browser`. As a result the bundle relies on some Node.js built-in like here in `dist/index.mjs`:

![Screenshot from 2023-05-03 09-55-31](https://user-images.githubusercontent.com/6571453/235862698-8e6474e2-98db-4ecf-a1bc-0e3ec0cdf59a.png)

This is why the Vercel deployments of https://github.com/Joystream/pioneer/pull/3872 were failing on Friday.
 
Because this pacakge doesn't have any production dependencies anymore, one way to fix that is to replace:
```suggestion
    "build": "tsup-node"
```
As described [here](https://tsup.egoist.dev/#excluding-all-packages).

However after testing this new build [directly on the Pioneer PR](https://github.com/Joystream/pioneer/pull/3872/commits/14693cc2ff51d5ca815d94f2d9c1b521ee615c54), icons break on the on boarding flow:

![image](https://user-images.githubusercontent.com/6571453/235865870-b4022317-6efc-4a09-a81b-db59bf8a87ca.png)

Also there is major issue (but I think it's caused by the proxy api not this package): When I confirm updating the metadata I get this error:
> Something went wrong with the query and rendering of this component. k.name is undefined

It completely breaks the polkadot wallet. The only way I found to sign transactions again is to remove the wallet and reinstall it so I lose all the accounts. It doesn't happen when disabling the proxy api so I think something in the metadata doesn't get serialized correctly.

![image](https://user-images.githubusercontent.com/6571453/235897967-f02672d6-5d00-4b5b-bd74-d3b61ebca5e2.png)